src/crimson/osd/pg_backend.cc

```diff
@@ -830,30 +830,32 @@ PGBackend::get_attr_ierrorator::future<> PGBackend::get_xattrs(
     return get_attr_errorator::now();
   });
 }
-static int do_xattr_cmp_str(int op, const string& lhs, bufferlist& rhs_xattr)
-{
-  string rhs(rhs_xattr.c_str(), rhs_xattr.length());
 
-  logger().debug("do_xattr_cmp_str '{}' vs '{}' op {}", lhs, rhs, op);
+namespace {
 
+template<typename U, typename V>
```

Comment:
~~Do we need `V`? Is there a case where it differs from `U`?~~

`string` and `string_view` in `PGBackend::cmp_xattr`.